source/renderer/app/App.tsx

```diff
@@ -65,46 +70,53 @@ class App extends Component<{
         <ThemeManager variables={themeVars} />
         <Provider stores={stores} actions={actions}>
           <AnalyticsProvider>
-            <MenuUpdater />
-            <ThemeProvider
-              theme={daedalusTheme}
-              skins={SimpleSkins}
-              variables={SimpleDefaults}
-              themeOverrides={themeOverrides}
-            >
-              <IntlProvider
-                {...{
-                  locale,
-                  key: locale,
-                  messages: translations[locale],
-                }}
-              >
-                <Fragment>
-                  <Router history={history}>
-                    <Routes />
-                  </Router>
-                  {mobxDevTools}
-                  {[
-                    // @ts-ignore ts-migrate(2345) FIXME: Argument of type 'string' is not assignable to par... Remove this comment to see the full error message
-                    isActiveDialog(ABOUT) && <AboutDialog key="aboutDialog" />,
-                    // @ts-ignore ts-migrate(2345) FIXME: Argument of type 'string' is not assignable to par... Remove this comment to see the full error message
-                    isActiveDialog(DAEDALUS_DIAGNOSTICS) && (
-                      <DaedalusDiagnosticsDialog key="daedalusDiagnosticsDialog" />
-                    ),
-                    // @ts-ignore ts-migrate(2345) FIXME: Argument of type 'string' is not assignable to par... Remove this comment to see the full error message
-                    isActiveDialog(TOGGLE_RTS_FLAGS_MODE) && (
-                      <ToggleRTSFlagsDialogContainer key="toggleRTSFlagsDialog" />
-                    ),
-                  ]}
-                  <RTSFlagsRecommendationOverlayContainer />
-                  <NotificationsContainer />
-                  {canShowNews && [
-                    <NewsFeedContainer key="newsFeedList" />,
-                    <NewsOverlayContainer key="newsFeedOverlay" />,
-                  ]}
-                </Fragment>
-              </IntlProvider>
-            </ThemeProvider>
+            <LocalStorageFeatureProvider localStorage={LocalStorageApi}>
```

Comment:
Relocated this from `index.tsx` file for consistency, and so DiscreetModeFeatureProvider can use AnalyticsProvider.